crates/ubrn_cli/fixtures/defaults/ubrn-wasm.rustflags.config.yaml

```diff
@@ -0,0 +1,6 @@
+rust:
+  directory: rust/shim
+  manifestPath: Cargo.toml
+
+wasm:
+  rustflags: ['--cfg', 'web_sys_unstable_apis', '-C', 'target-feature=+atomics']
```

Comment:
do you think we should think about (not in this PR) having `rustflags` wherever you can add `cargoExtras`?

Reply:
Yeah, that's probably a good idea.